meta-balena-common/recipes-core/images/balena-image.bb

```diff
@@ -59,14 +59,9 @@ generate_hostos_version () {
 }
 
 symlink_image_signature () {
-    # This is probably not the correct way to do it, but it works.
-    # We sign BALENA_RAW_IMG, which ends up in IMGDEPLOYDIR
-    # and has a timestamp in the file name. We need to get rid
-    # of the timestamp for the final deploy, so that the file
-    # ends up in a predictable location.
-
     if [ -n "${SIGN_API}" ]; then
-        ln -sf "${BALENA_RAW_IMG}.sig" "${DEPLOY_DIR_IMAGE}/balena-image-${MACHINE}.balenaos-img.sig"
+        ln -sf "${IMAGE_NAME}.rootfs.balenaos-img.sig" \
+            "${DEPLOY_DIR_IMAGE}/balena-image-${MACHINE}.balenaos-img.sig"
```

Comment:
I don't understand why `BALENA_RAW_IMG` is not working though:
```
alex@balenetxe /build/balena-iot-gate-imx8/build (alexgg/hab)$ MACHINE=iot-gate-imx8 bitbake balena-image -e | grep ^BALENA_RAW_IMG=
BALENA_RAW_IMG="/build/balena-iot-gate-imx8/build/tmp/work/iot_gate_imx8-poky-linux/balena-image/1.0-r0/deploy-balena-image-image-complete/balena-image-iot-gate-imx8-20250111111959.rootfs.balenaos-img"
```

Reply:
@alexgg It looks like the difference is that `${BALENA_RAW_IMAGE}` is the path of the signature in the work directory, whereas this PR symlinks to the deployed signature in the images directory.